src/base1/cockpit.js

```diff
@@ -3865,6 +3866,7 @@ function factory() {
         }
 
         cockpit.language = lang;
+        cockpit.js_lang = lang.replace('_', '-')
```

Comment:
The commit message also says that the country code should be lower case. Do we need a `.toLowerCase()` here?

Reply:
it's case-insensitive

Reply:
in fact, all of the examples on MDN are uppercase